scripts/mkimage

```diff
@@ -40,8 +40,12 @@ cleanup() {
 
 show_error() {
   echo -e "image: error happen...\n"
-  cat "${SAVE_ERROR}"
-  echo
+  if [ -s "${SAVE_ERROR}" ]; then
+    cat "${SAVE_ERROR}"
+    echo
+  else
+    echo -e "  folder $LE_TMP might be out of free space...\n"
+  fi    
```

Comment:
How about:
```
  if [ -s "${SAVE_ERROR}" ]; then
    cat "${SAVE_ERROR}"
  else
    echo "Folder $LE_TMP might be out of free space..."
  fi
  echo
````

Combined with the change above, this makes the vertical spacing a little more obvious.